app/messenger/MainMessenger.js

```diff
@@ -106,8 +107,14 @@ class MainMessenger {
     this.contents = { ...props };
     const storage = new Storage();
     const install = storage.get('install');
-    if (!install) {
-      this.initializeInstallerWindow();
+    const wslConfigured = storage.get('wslConfigured');
+    let wsl2Supported = false;
```

Comment:
you can make this a const
booleans should be recognizable with is or are as a prefix 
```suggestion
    const build = os.release().split('.')[2];
    const isWsl2Supported = isWindows &&  (Number(build) >= 19041);
```